osu.Game/Graphics/UserInterface/Volume/VolumeControl.cs

```diff
@@ -87,9 +111,34 @@ private void volumeChanged(double newVolume)
             schedulePopOut();
         }
 
+        private readonly BindableDouble muteBindable = new BindableDouble();
+
+        public bool IsMuted { get; private set; }
+
+        public void Mute()
+        {
+            if (IsMuted)

```

Comment:
i'd combine the content of Mute and Unmute into a single (private?) method ie. `setMute(bool muted)` or just use a `BindableBool` instead.

Reply:
I changed the Mute and Unmute methods to just change the value of BindableBool and then mute the volume via it's ValueChanged action. I hope that's what you were thinking of.